.github/workflows/clamav-release.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024, Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+name: Scan release archives with ClamAV
+
+permissions: read-all
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'ISPC release version (just number without v, e.g., 1.24.0)'
+        required: true
+        type: string
+
+env:
+  release_base_url: https://github.com/ispc/ispc/releases/download/
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install ClamAV
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clamav
+
+    - name: Update ClamAV database
+      run: |
+        sudo systemctl stop clamav-freshclam
+        sudo freshclam
+        sudo systemctl start clamav-freshclam
+
+    - name: Download and unpack release artifacts
+      env:
+        V: ${{ inputs.version }}
+      run: |
+        TAR="ispc-examples-v$V.tar.gz ispc-v$V-linux-oneapi.tar.gz \
+             ispc-v$V-linux.aarch64.tar.gz ispc-v$V-linux.tar.gz \
+             ispc-v$V-macOS.arm64.tar.gz ispc-v$V-macOS.universal.tar.gz \
+             ispc-v$V-macOS.x86_64.tar.gz"
+        ZIP="ispc-examples-v$V.zip ispc-v$V-windows.zip"
+        for f in $TAR $ZIP; do
+          wget -O $f ${{ env.release_base_url }}/v$V/$f
+        done
+        mkdir zip tar
+        for f in $ZIP; do
```

Comment:
It looks like that ClamAV can scan archives `--scan-archive --archive-verbose` so probably no need to unpack them?
https://manpages.ubuntu.com/manpages/jammy/man1/clamscan.1.html

Reply:
I have added it, but it doesn't seem to be reliable enough for me. It doesn't scan all archives by some reason ([link](https://github.com/nurmukhametov/ispc/actions/runs/10216148461)).

Reply:
OK